triggers/runner/src/main/scala/com/digitalasset/daml/lf/engine/trigger/Runner.scala

```diff
@@ -244,6 +277,25 @@ object Trigger extends StrictLogging {
       case v => Left(s"Expected AllInDar or RegisteredTemplates but got $v")
     }
   }
+
+  sealed abstract class Version extends Serializable
+  object Version {
+    case object `2.0` extends Version
+    case object `2.5` extends Version
```

Comment:
are these values hinting that the trigger version will be tied to the SDK version? I think I'd rather go with `0.1` and `1.0`

Reply:
my bad, (even though I said trigger above) I assumed this was for daml-_script_; since triggers live beyond the scenario service it does make sense to share the version with the SDK.